library/src/main/java/io/ona/kujaku/downloaders/MapBoxOfflineResourcesDownloader.java

```diff
@@ -418,7 +437,10 @@ public void pauseMapDownload(final String name, final OnPauseMapDownloadCallback
             public void onList(OfflineRegion[] offlineRegions) {
                 final OfflineRegion offlineRegion = getOfflineRegion(name, offlineRegions);
                 if (offlineRegion == null) {
-                    onPauseMapDownloadCallback.onPauseError(OnPauseMapDownloadCallback.MAP_COULD_NOT_BE_FOUND, "");
+                    Log.e(TAG, "Map could not be found");
+                    if (onPauseMapDownloadCallback != null) {
+                        onPauseMapDownloadCallback.onPauseError(OnPauseMapDownloadCallback.MAP_COULD_NOT_BE_FOUND, "");
```

Comment:
Same question applies to this listener

Reply:
I could change this to have only one string returned explaining the error since I can't seem to find why I put it there